### DISCUSSION TITLE
This is a question about two factor authentication.

### DISCUSSION THREAD

**Discussion by soonee** [community/community#28841] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello, I am a github user and my username is soonee [Community Manager edited to remove PII (user email)]

There seems to be an error in the 2FA function.

I recently purchased an additional iPhone 13 Pro Max and factory reset my iPhone 8 Plus device.
And I signed in with my Apple ID to the iPhone 13 Pro Max device.
The device we are currently using is the iPhone 13 Pro Max.

On the github website, two devices are registered with 2FA.

However, it seems that the sudo mode is only pushed to the iPhone 8 Plus device.
So, all github services related to sudo mode are unavailable.

In summary, what I am asking for is to delete all 2FA devices, or only the iPhone 8 Plus devices.

If that's hard, can you make the push go to both devices?

Please answer fast.

**Comment by airtower-luna** [comment, marked as answer]
You'll have to [contact GitHub support](https://support.github.com/request) about account details, us other users can't help there.

> If that's hard, can you make the push go to both devices?

Are you using SMS (in that case there is a push, but it should depend only on your SIM), or TOTP? In case of TOTP there is no push, it's time combined with a secret stored in your phone.

Hopefully you have recovery codes, a token, or SSH key just in case you need to recover the account at some point. Good luck!

**Reply by soonee** [reply]
This is the service I wanted, on GitHub.
thank you.
